.github/workflows/image-cleanup.yml

```diff
@@ -9,20 +9,24 @@ jobs:
   delete-images:
     name: Delete PR-scoped test images
     runs-on: ubuntu-latest
-    steps:
-      - name: Check out code
-        uses: actions/checkout@v2
 
+    strategy:
+      matrix:
+       image-names:
```

Comment:
Lets use it in array form and call it `matrix.image`

```yaml
       image: [cryostat-operator, cryostat-operator-bundle, cryostat-operator-scorecard]
````

Reply:
yah this looks cleaner